src/ontology/uberon-edit.obo

```diff
@@ -221904,19 +221904,20 @@ relationship: part_of UBERON:6001767 ! insect antennal disc
 
 [Term]
 id: UBERON:7500033
-name: distal-most point of medial condyle of femur
-def: "The distal-most point of the medial condyle of a femur." [https://orcid.org/0000-0003-2699-3066]
-intersection_of: UBERON:0006983 ! anatomical point
-intersection_of: distalmost_part_of UBERON:0009984 ! medial condyle of femur
-relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
+name: obsolete distal-most point of medial condyle of femur
+def: "OBSOLETE The distal-most point of the medial condyle of a femur." [https://orcid.org/0000-0003-2699-3066]
+comment: too specific of a term
+is_obsolete: true
+replaced_by: UBERON:0009984
 
 [Term]
 id: UBERON:7500034
-name: distal-most point of trochlea of humerus
-def: "The distal-most point of the trochlea of a humerus." [https://orcid.org/0000-0003-2699-3066]
-intersection_of: UBERON:0006983 ! anatomical point
-intersection_of: distalmost_part_of UBERON:0000144 ! trochlea of humerus
+name: obsolete distal-most point of trochlea of humerus
```

Comment:
looks like you did obsolete UBERON:7500034

Reply:
oops! ok I'll fix it.